package.json

```diff
@@ -28,8 +28,9 @@
     "@beyonk/svelte-mapbox": "^8.1.4",
     "@fortawesome/free-solid-svg-icons": "^6.1.1",
     "@googlemaps/js-api-loader": "^1.13.10",
-    "@mapbox/mapbox-gl-geocoder": "^4.7.4",
```

Comment:
This was redundant it seems because of `@beyonk/svelte-mapbox`. Let me know if I've misunderstood why this is here though!